packages/opc-comment-input/README.md

```diff
@@ -0,0 +1,69 @@
+# opc-comment-input 👋
+
+![Version](https://img.shields.io/badge/version-0.0.1-blue.svg?cacheSeconds=2592000)
+![Build Status](https://travis-ci.org/dwyl/esta.svg?branch=master)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/1-Platform/op-components/graphs/commit-activity)
+
+A standardized web component based on Lit Element for Red Hat One Platform to use as a comment input component.
+
+## Prerequisites
+The `opc-comment-input` component is dependent on the Patternfly library for styling. So to avoid any missing styles, add the `patternfly.css` file before the component script tags.
+
+# Usage
```

Comment:
@gisumit Can you use the newer format which has examples for Angular and React

Reply:
yeah, added usage for angular, react, and vanilla js in the README.md file.

Reply:
can you review it again?
@diwanshi